register_commands.py

```diff
@@ -0,0 +1,35 @@
+import requests
+
+APPLICATION_ID = 154
+BOT_TOKEN = "154"
+
+url = "https://discord.com/api/v8/applications/{}/commands".format(APPLICATION_ID)
+
+
+# This is an example CHAT_INPUT or Slash Command, with a type of 1
+json = {
+    "name": "dldialogue",
+    "type": 1,
+    "description": "Create a Dragalia Lost themed dialogue",
+    "options": [
+        {
+            "name": "name",
+            "description": "The name of the character",
+            "type": 3,
+            "required": True
+        },
+        {
+            "name": "message",
+            "description": "What should the character say",
+            "type": 3,
+            "required": True
+        },
+    ]
+}
+
+# For authorization, you can use either your bot token
+headers = {
+    "Authorization": "Bot {}".format(BOT_TOKEN)
+}
+
+r = requests.post(url, headers=headers, json=json)
```

Comment:
Why is this a part of this project? All of this feels like a client to the DLDialogue API server. Not every client is going to be incorporated directly into the server code.

This file specifically feels like a separate project, or at least an "example" that should go Ina separate folder. I know there's some issues with https to; it almost feels like the `/discord` endpoint should really be its own service on https that makes requests to this API.